vignettes/data-combined.Rmd

```diff
@@ -0,0 +1,300 @@
+can easily achieve this with the scale factor. In the next example, we normalize 
+observed values to 250 mg dose by setting the `yScaleFactor` to 1/250:
+
+```{r}
+myDataCombinedTranformations$setDataTransformations(forNames = obsData$`Vergin 1995.Iv`$name,
+                                      yScaleFactors = 1/250)
+
+plotIndividualTimeProfile(dataCombined = myDataCombinedTranformations)
+```
+Finally, offsetting the observation values might be useful when working with measurements 
+of endogenous substrates, such as the hormone glucogen, and want to correct for the 
+individual specific baseline levels of the hormone.
+
+## Extracting a combined data frame
+
+The data frame (also sometimes called as a table) data structure is central to R-based workflows, and, thus, we may wish to extract a data frame for datasets present in the object.
+
+Internally, `DataCombined` extracts data frames for observed and simulated datasets and combines them.
+
+```{r}
+myDataCombined$toDataFrame()
+```
+
+This function returns a [tibble data frame](https://r4ds.had.co.nz/tibbles.html#tibbles-vs.-data.frame). If you wish to modify how it is printed, you can have a look at the available options [here](https://pillar.r-lib.org/reference/pillar_options.html). In fact, let's change a few options and print the data frame again.
+
+```{r, eval=FALSE}
+options(
+  pillar.width = Inf, # show all columns
+  pillar.min_chars = Inf # to turn off truncation of column titles
+)
+
+myDataCombined$toDataFrame()
+```
+
+
+```{r, echo=FALSE}
+# change these settings only temporarily
+withr::with_options(
+  list(
+    pillar.width = Inf, # show all columns
+    pillar.min_chars = Inf # to turn off truncation of column titles
+  ),
+  code = {
+    print(myDataCombined$toDataFrame())
+  }
+)
+```
+
+As can be seen, this data frame attemtps to provide a one-stop access to all important details about the datasets included in the object, including important meta-data.
+
+`{ospsuite}` also provides a few helper functions to modify the data frame further.
+
+When multiple (observed and/or simulated) datasets are present in a data frame, they are likely to have different units. `convertUnits()` function helps to convert them to a common unit^[Note that if you are using `DataCombined` object for plotting functions, you don't need to this conversion; the plotting functions will take care of this internally.].
+
+```{r}
+convertUnits(
+  myDataCombined,
+  xUnit = ospUnits$Time$s,
+  yUnit = ospUnits$`Concentration [mass]`$`µg/l`
+)
+```
+
+## Further functionalities
+
+Grouping of simulated results with observed data inside `DataCombined` also allows 
+to calculate the error between simulated results and observations using the 
+`calculateResiduals()` function. The function calculates the distance between 
+each observed data point within the group and the simulated results. Simulation 
+results interpolated if no simulated value is available for a certain observed 
+time point.
```

Comment:
Observed data beyond the simulation range is ignored, right? Should be added here.

Reply:
> The function calculates the distance between 
each observed data point within the group and the simulated results. Simulation 
results interpolated if no simulated value is available for a certain observed 
time point. Observed data beyond simulated time is ignored, i.e., no *extrapolation* 
is performed.